native/src/components/SettingsSwitch.tsx

```diff
@@ -3,7 +3,7 @@ import { Switch } from 'react-native'
 import { useTheme } from 'styled-components'
 
 type SettingsSwitchProps = {
-  onPress: () => void
+  onPress: (value: boolean) => void
```

Comment:
I guess we should update the other uses of SettingsSwitch then?

Reply:
Not needed, its basically just an optional parameter.